app/views/likes/create.js.erb

```diff
@@ -0,0 +1,3 @@
+$("#likes_<%= @recipe.id %>").html('<%= @recipe.likes.count %>&nbsp;');
+$("#like-word_<%= @recipe.id %>").html("<%= @recipe.likes.count == 1 ? 'like' : 'likes' %>");
```

Comment:
use `pluralize` here from Rails instead of the ternary

Reply:
done!